sentinelsat/scripts/cli.py

```diff
@@ -217,7 +229,24 @@ def cli(
         search_kwargs["area"] = wkt
 
     if geometry is not None:
-        search_kwargs["area"] = geojson_to_wkt(read_geojson(geometry))
+        # check if the value is an existing path
+        if os.path.exists(geometry):
+            search_kwargs["area"] = geojson_to_wkt(read_geojson(geometry))
+        # check if the value is a GeoJSON
+        else:
+            try:
+                geometry = eval(geometry)
```

Comment:
Using `eval()` is not a good practice in general, since you allow any syntactically valid Python to be executed, which can have a lot of unintended consequences (usually security related, which is not as big of a concern for a CLI utility, but still). Use `json.loads()` instead.
Also, instead of relying on parsing errors to detect non-GeoJSON content, I would check whether `geometry.strip()` starts with `{`. That way any accidental JSON errors would not get ignored.

Reply:
Good call, I'll keep the `eval()` in mind for the future.
I've kept the parsing try except in case there are any errors inside the json string that would make it invalid, but added the check for `{`.